internal/container/controller.go

```diff
@@ -203,6 +224,84 @@ func (c Controller) ContainerFiles(id string, filespec string) (files []string)
 	return strings.Split(string(stdout), "\n")
 }
 
+func (c Controller) DownloadFile(id string, src string, dest string) {
+	if id == "" {
+		panic("Must pass in non-empty id")
+	}
+	if src == "" {
+		panic("Must pass in non-empty src")
+	}
+	if dest == "" {
+		panic("Must pass in non-empty dest")
+	}
+
+	cmd := []string{"mkdir", "/var/opt/sql/backup"}
```

Comment:
>"/var/opt/sql/backup" [](http://example.com/codeflow?start=26&length=21)

dest ?

Reply:
/var/opt/sql/backup is the well known folder to place .bak files in. (this is a common pattern in blog posts etc.